### ISSUE TITLE
Design of the power grid

### ISSUE THREAD

**Issue by SaulApostle** [Regalis11/Barotrauma#1680]
- [x] I have searched the issue tracker to check if the issue has already been reported.
I searched for the posts I had, but my English level was limited, and I wasn't sure they were the same thing as I was talking about.

There are some problems with the design of the power grid for this game.

Scenario a
Battery and bulb connection
Battery and junction box connection
junction box and bulb connection
Bulb Power Consumption 1
junction box power 2
Batteries Reduce Double Electricity

![T}}7AWBNOF($}U86}J{WJIE](https://user-images.githubusercontent.com/11502419/59974220-2d7cad00-95dc-11e9-9a15-681bff3312dd.png)

Scenario b
Battery and junction box A connection
junction box A and bulb connection
junction box B and bulb connection
The light bulb does not work
Batteries are being depleted

![image](https://user-images.githubusercontent.com/11502419/59973208-010e6400-95cf-11e9-8913-2c108f9886a7.png)

Simply put,No branch lines are allowed in the power grid
,Nor are two wires allowed to connect to the same appliance.

I hope developers can refer to the game Oxygen Not Included,Now the grid is buggy and untrue.

**Version**
9.0.6

**Comment by hubismal** [comment]
do you think you can send pictures of your described scenarios?

**Comment by SaulApostle** [comment]
> do you think you can send pictures of your described scenarios?

No pictures are allowed to be uploaded here.

**Comment by thegrb93** [comment]
That's not true

**Comment by Rokvach** [comment]
Tested, seems to be working correctly. Closing.
